### ISSUE TITLE
Error initializing firewall on OpenWrt Snapshot

### ISSUE THREAD

**Issue by acimanx** [openNDS/openNDS#302]
Device: Linksys WRT32X
Firmware: OpenWrt SNAPSHOT r18776-09f383465e
Kernel Version: 5.10.96

opennds 9.5.1-1
dnsmasq	2.86-11
dnsmasq-full	2.86-11

I'm folowing instruction to install openNDS on openwrt https://opennds.readthedocs.io/en/stable/install.html
```
opkg install opennds
reboot

```
After reboot there is no splash page for devices.
Trying logread and see there is an openNDS bootlop with repeating 

```
dnsmasq: exiting on receipt of SIGTERM
OpenNDS Error initializing firewall

```
After that I've tried to switch between dnsmasq and dnsmasq-full. But got the same result.

**Comment by bluewavenet** [comment]
@acimanx 
Yes, this is an issue with current snapshot. It is related to a problem with iptables-nft which is giving an error in snapshot.
If you revert to the release version of OpenWrt, all will be well as it will then use iptables-legacy.
A fix for snapshot will come in due course.

**Comment by acimanx** [comment]
Thanks for your reply. I thought I missing something in documentation. 
I couldn't find any information about this problem in iptables in whole internet.

**Comment by bluewavenet** [comment]
@acimanx 
I now have a work around for the problem in testing. I am in UTC time zone  and it is late so will post details tomorrow.

**Comment by bluewavenet** [comment]
@acimanx 
This might work: use opkg to remove iptables, selecting "remove dependent packages" when warned.

Then install:
iptables-nft
kmod-ipt-conntrack
kmod-ipt-nat

I will have detailed instructions later, but please post how you get on.

**Comment by bluewavenet** [comment]
@acimanx 
This is now fixed with the release of v9.6.0
